securedrop/source_templates/lookup.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% block body %}
 
-<center>
+<div style="text-align: center;">
```

Comment:
SecureDrop is served with `Content-Security-Policy: default-src 'self'`, which blocks inline styles (see the Apache conf for the source interface in `install_files/ansible-base/roles/app/templates/sites-available/source.conf`). It would be better to specify this with a class or id, and define the style rules in `/static/css/styles.css`. There is already a general-purpose `.center` class that will do exactly what you want.